ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -631,10 +631,12 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FILTERS_ADDITIONAL: ""
       FIPS_ENABLED: "true"
+      INDEX_IMAGE: quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-12@sha256:42bbbc8723e1ab7a4a6b71bfbf2f47f0a8e41aab9596f97c09a7f751705d3ab4
```

Comment:
Should be able to update this to:

[quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-12@sha256:0aa0883f073a4bf86eed59fe916c9bdbe738da903d1e556a70dfdaca9e35b103](http://quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-12@sha256:0aa0883f073a4bf86eed59fe916c9bdbe738da903d1e556a70dfdaca9e35b103)